src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/recordings/test_mysql_flexible_server_byok_mgmt.yaml

```diff
@@ -0,0 +1,3489 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - keyvault create
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - -g -n --location --enable-soft-delete --enable-purge-protection
+      User-Agent:
+      - AZURECLI/2.38.0 azsdk-python-azure-mgmt-keyvault/9.3.0 Python/3.8.10 (Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.29)
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.KeyVault/vaults/rdbmsvault000002?api-version=2021-06-01-preview
```

Comment:
Wanted to make sure you notice that test recordings mask the resource names like this. In some cases where you need the exact names for the resources in the record runs, you will need to set it as @liveonly. Please see tests for public access network setting. You're good to just resolve this comment.